### PR TITLE
feat: add AudioCLIPTextEncoder

### PR DESCRIPTION
**Pull request by tadejsv**

### CODE REVIEW

File: jinahub/encoders/text/AudioCLIPTextEncoder/README.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+```python
+from jina import Flow, Document
+
+f = Flow().add(
+	uses='jinahub+docker://AudioCLIPTextEncoder',
+	volumes='/path/to/pwd/.cache:/workspace/.cache'
+)
+
+with f:
+	doc = Document(text='test text')
```
Comment (bwanglzu) [inline jinahub/encoders/text/AudioCLIPTextEncoder/README.md:146]: 4 extra spaces here

File: jinahub/encoders/text/AudioCLIPTextEncoder/tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+@pytest.fixture(scope="session", autouse=True)
+def download_cache():
+    os.system('scripts/download_full.sh')
+    yield
+    shutil.rmtree('.cache') 
+
+
+@pytest.fixture()
+def test_dir() -> str:
+    return os.path.dirname(os.path.abspath(__file__))
```
Comment (bwanglzu) [inline jinahub/encoders/text/AudioCLIPTextEncoder/tests/conftest.py:21]: this fixture seems not being used

**Review by bwanglzu — CHANGES_REQUESTED** [review]
Great work! left some minor comments

File: jinahub/encoders/text/AudioCLIPTextEncoder/README.md (hunk 1 above)
Comment (tadejsv) [reply jinahub/encoders/text/AudioCLIPTextEncoder/README.md:146]: It just looks like that - it is actually a single tab. I will replace the tab with 4 spaces here then

File: jinahub/encoders/text/AudioCLIPTextEncoder/tests/conftest.py (hunk 2 above)
Comment (tadejsv) [reply jinahub/encoders/text/AudioCLIPTextEncoder/tests/conftest.py:21]: Will remove

**Review by bwanglzu — APPROVED** [review]
LGTM!

**Pull request merged**